recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - libboost-headers
     - libitk-devel
   run:
-    - libitk
+    - {{ pin_compatible('libitk', min_pin='x.x', max_pin='x.x') }}
```

Comment:
What is this doing? I am not familiar with this feature.

Reply:
https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#pinning-at-the-recipe-level

This uses the version of the package present at build, and sets version constraints. Since 5.4.0 is installed, this becomes `libitk >=5.4,<5.5`.

When 6.0 is used, it will auto update to `libitk >=6.0,<6.1`.

Reply:
this should not be necessary for you to think about, it should be resolved at the libitk level:
https://github.com/conda-forge/libitk-feedstock/issues/92